CMakeLists.txt

```diff
@@ -139,7 +139,17 @@ if (NGRAPH_ONNX_IMPORT_ENABLE)
     option(NGRAPH_ONNXIFI_ENABLE "Enable ONNX Interface for Framework Integration" TRUE)
 endif()
 
-option(NGRAPH_PLAIDML_ENABLE "Enable the PlaidML backend" FALSE)
+option(NGRAPH_PLAIDML_ENABLE "Enable the PlaidML backend" ON)
+
+if (NGRAPH_PLAIDML_ENABLE)
+    find_package(PlaidML CONFIG)
+    if (NOT PLAIDML_FOUND)
+        message(STATUS "PlaidML not installed; disabling PlaidML backend")
```

Comment:
If the user specifies to build PlaidML then it should either build it or fail. In the past we had this sort of option for the GPU backend and for a couple of weeks our CI was not testing the GPU backend and was passing even though the GPU backend was requested for the build.

Reply:
Something like this?